tests/unit_tests/example_test/Makefile.am

```diff
@@ -0,0 +1,13 @@
+AUTOMAKE_OPTIONS = foreign
+
+check_PROGRAMS = testdriver testdriver2
```

Comment:
Yes, or testdriver_example1, so that you can still use your catch-all in .gitignore.  I was more talking in general terms, as the auth-pam test was also called 'testdriver'.